src/expr/src/relation/mod.rs

```diff
@@ -1298,14 +1297,6 @@ impl MirRelationExpr {
         .unwrap()
     }
 
-    /// Pretty-print this MirRelationExpr to a string.
```

Comment:
These `pretty_*` ones are handy when doing `println`-based debugging of the transforms.

Reply:
thanks for getting through them! re-added